tools/libSwiftSyntaxParser/CMakeLists.txt

```diff
@@ -10,14 +10,24 @@ set(LLVM_EXPORTED_SYMBOL_FILE
 add_swift_host_library(libSwiftSyntaxParser SHARED
   c-include-check.c
   libSwiftSyntaxParser.cpp)
+
+add_swift_host_library(libSwiftSyntaxParserStatic STATIC
```

Comment:
I don't feel great about all this duplication, but also since only ~half of the things here applied to this, I went with it over creating a function. I'm not a cmake expert so I'd appreciate input!